src/main/java/cc/woverflow/hytils/mixin/GuiIngameForgeMixin_HideHotbar.java

```diff
@@ -0,0 +1,198 @@
+        if (HytilsConfig.hideHudElements && locraw != null && HypixelUtils.INSTANCE.isHypixel()) {
+            if (HytilsReborn.INSTANCE.getLobbyChecker().playerIsInLobby()) {
+                ci.cancel();
+                return;
+            }
+
+            LocrawInfo.GameType gameType = locraw.getGameType();
+            String gameMode = locraw.getGameMode();
+            switch (gameType) {
+                case TNT_GAMES:
+                    // break out of the switch if it's a tnt game that has varying hunger
+                    if (gameMode.contains("CAPTURE")) {
+                        break;
+                    }
+                case BEDWARS:
+                case MURDER_MYSTERY:
+                case HOUSING:
+                case LIMBO:
+                case PAINTBALL:
+                case PIT:
+                case DUELS:
+                case BUILD_BATTLE:
+                case QUAKE:
+                case REPLAY:
+                case WOOL_WARS:
+                    ci.cancel();
+                    return;
+            }
+
+            switch (gameMode) {
+                case "PIXEL_PARTY":                 // pixel party (arcade)
+                case "PIXEL_PARTY_HYPER":           // pixel party (arcade)
+                case "PVP_CTW":                     // capture the wool (arcade)
+                case "ZOMBIES_DEAD_END":            // zombies (arcade)
+                case "ZOMBIES_BAD_BLOOD":           // zombies (arcade)
+                case "ZOMBIES_ALIEN_ARCADIUM":      // zombies (arcade)
+                case "HIDE_AND_SEEK_PROP_HUNT":     // hide and seek (arcade)
+                case "HIDE_AND_SEEK_PARTY_POOPER":  // hide and seek (arcade)
+                case "MINI_WALLS":                  // miniwalls (arcade)
+                case "STARWARS":                    // galaxy wars (arcade)
+                case "HOLE_IN_THE_WALL":            // hole in the wall (arcade)
+                case "SOCCER":                      // football (arcade)
+                case "ONEINTHEQUIVER":              // bounty hunters (arcade)
+                case "DRAW_THEIR_THING":            // pixel painters (arcade)
+                case "ENDER":                       // ender spleef (arcade)
+                case "DROPPER":
+                    ci.cancel();
+            }
+        }
+    }
+
+    @Inject(method = "renderArmor", at = @At("HEAD"), cancellable = true)
+    public void cancelArmor(int width, int height, CallbackInfo ci) {
+        LocrawInfo locraw = LocrawUtil.INSTANCE.getLocrawInfo();
+        if (HytilsConfig.hideHudElements && locraw != null && HypixelUtils.INSTANCE.isHypixel()) {
+            if (HytilsReborn.INSTANCE.getLobbyChecker().playerIsInLobby()) {
+                ci.cancel();
+                return;
+            }
+
+            LocrawInfo.GameType gameType = locraw.getGameType();
+            String gameMode = locraw.getGameMode();
+            switch (gameType) {
+                case DUELS:
```

Comment:
if the logic for all of this is the same (haven't seen the actual code too lazy to + on ipad) can we move this to on method